### PR TITLE
Update uuid version

### PR DESCRIPTION
**Pull request by athos-ribeiro**
Signed-off-by: Athos Ribeiro <athos.ribeiro@canonical.com>

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -17,13 +17,13 @@ clap = { version = "3.1.18", features = ["derive"]}
 env_logger = "0.8.3"
 log = "0.4"
 serde_json = {version = "1.0", features = ["preserve_order"]}
-uuid = {version = "0.8.2", features = ["v4"]}
+uuid = {version = "1.1.2", features = ["v4"]}
```
Comment (jonner) [inline Cargo.toml:20]: I think for now I'd rather specify this as "1.0". This should allow any version 1.x

**Review by jonner — CHANGES_REQUESTED** [review]
Thanks for the contribution. It's helpful if you run 'cargo fmt' before committing to ensure it follows the coding style. Let me know if you'd like me to fix it up and resubmit (with the version spec change). Otherwise I can do it for you. Thanks!

**Comment by athos-ribeiro** [comment]
Hi Jonathon,

Thanks for the feedback!

I performed the requested change. I deliberately did not change the linter issue in the workflow to keep this PR to a minimal (it pre-dates this PR). If you want, I can add an additional commit to address that.

**Comment by jonner** [comment]
Hi Athos, I wasn't talking about a lint issue, but a code formatting issue. I use `cargo fmt` as the final authority on code formatting since it does a pretty good job and makes my life easier. When changing from `to_hyphenated()` to `hyphenated()` in `MDev::undefine()`, the line length shrinks enough so that it can now all fit on a single line. Because of this, `cargo fmt` reformats it to a single line. There's no need for a separate commit to change that, since it's the result of the changes in this patch.

**Comment by athos-ribeiro** [comment]
Fair enough. Changed!

Thank you :)

**Comment by jonner** [comment]
Excellent, thank you! Congratulations on your first contribution to mdevctl.

**Pull request merged**